### PR TITLE
Orphanet slurp - add new terms

### LINKED ISSUE #2279: Orphanet slurp
I just ran the Orphanet slurp yesterday (@cmungall), and this is what it came up with:

I don't know what to do with this; @nicolevasilevsky maybe you can figure it out? 

```
xref:  {source="MONDO:equivalentTo"} ! 

[Term]
id: MONDO:0034846
name: primary desmosis coli
subset: ordo_disease
xref: ICD10:K59.9 {source="Orphanet:565641"} {http://www.geneontology.org/formats/oboInOwlsource="NTBT (ORPHA code's Narrower Term maps to a Broader Term)"}
is_a: MONDO:0015245 {source="Orphanet:565641"} ! obsolete rare intestinal disease
is_a: MONDO:0000001 {source="Orphanet:565641"} ! disease or disorder
xref: Orphanet:565641 {source="MONDO:equivalentTo"} ! primary desmosis coli

[Term]
id: MONDO:0034849
name: laminin subunit alpha 2-related limb-girdle muscular dystrophy r23
subset: ordo_disease
is_a: MONDO:0015152 {source="Orphanet:565837"} ! autosomal recessive limb-girdle muscular dystrophy
is_a: MONDO:0016149 {source="Orphanet:565837"} ! qualitative or quantitative defects of merosin
is_a: MONDO:0000001 {source="Orphanet:565837"} ! disease or disorder
xref: Orphanet:565837 {source="MONDO:equivalentTo"} ! laminin subunit alpha 2-related limb-girdle muscular dystrophy r23

[Term]
id: MONDO:0034965
name: middle east respiratory syndrome
subset: ordo_disease
is_a: MONDO:0015118 {source="Orphanet:576074"} ! obsolete rare pulmonary disease
is_a: MONDO:0015576 {source="Orphanet:576074"} ! obsolete rare viral disease
is_a: MONDO:0000001 {source="Orphanet:576074"} ! disease or disorder
xref: Orphanet:576074 {source="MONDO:equivalentTo"} ! middle east respiratory syndrome

[Term]
id: MONDO:0034972
name: nlrc4-related familial cold autoinflammatory syndrome
subset: ordo_disease
xref: OMIM:616115 {source="Orphanet:576349"} {http://www.geneontology.org/formats/oboInOwlsource="E (Exact mapping: the two concepts are equivalent)"}
is_a: MONDO:0017370 {source="Orphanet:576349"} ! autoinflammatory syndrome with skin involvement
is_a: MONDO:0017953 {source="Orphanet:576349"} ! hereditary periodic fever syndrome
is_a: MONDO:0000001 {source="Orphanet:576349"} ! disease or disorder
relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/16412 {source="Orphanet:576349"} ! NLRC4
xref: Orphanet:576349 {source="MONDO:equivalentTo"} ! nlrc4-related familial cold autoinflammatory syndrome

[Term]
id: MONDO:0034975
name: variant creutzfeldt-jakob disease
subset: ordo_disease
is_a: MONDO:0000001 {source="Orphanet:576370"} ! disease or disorder
is_a: MONDO:0018686 {source="Orphanet:576370"} ! acquired Creutzfeldt-Jakob disease
xref: Orphanet:576370 {source="MONDO:equivalentTo"} ! variant creutzfeldt-jakob disease

[Term]
id: MONDO:0034976
name: iatrogenic creutzfeldt-jakob disease
subset: ordo_disease
is_a: MONDO:0000001 {source="Orphanet:576379"} ! disease or disorder
is_a: MONDO:0018686 {source="Orphanet:576379"} ! acquired Creutzfeldt-Jakob disease
xref: Orphanet:576379 {source="MONDO:equivalentTo"} ! iatrogenic creutzfeldt-jakob disease

[Term]
id: MONDO:0034987
name: intraductal tubulopapillary neoplasm of pancreas
subset: ordo_disease
is_a: MONDO:0000001 {source="Orphanet:580572"} ! disease or disorder
is_a: MONDO:0018520 {source="Orphanet:580572"} ! rare epithelial tumor of pancreas
xref: Orphanet:580572 {source="MONDO:equivalentTo"} ! intraductal tubulopapillary neoplasm of pancreas

[Term]
id: MONDO:0035004
name: serine biosynthesis pathway deficiency, infantile/juvenile form
subset: ordo_disease
is_a: MONDO:0015162 {source="Orphanet:583595"} ! obsolete rare syndromic intellectual disability
is_a: MONDO:0015983 {source="Orphanet:583595"} ! obsolete rare genetic syndromic intellectual disability
is_a: MONDO:0018162 {source="Orphanet:583595"} ! neurometabolic disorder due to serine deficiency
is_a: MONDO:0000001 {source="Orphanet:583595"} ! disease or disorder
xref: Orphanet:583595 {source="MONDO:equivalentTo"} ! serine biosynthesis pathway deficiency, infantile/juvenile form

[Term]
id: MONDO:0035008
name: isolated splenic vein thrombosis
subset: ordo_disease
is_a: MONDO:0000001 {source="Orphanet:583856"} ! disease or disorder
is_a: MONDO:0019997 {source="Orphanet:583856"} ! obsolete rare gastroenterologic disease
xref: Orphanet:583856 {source="MONDO:equivalentTo"} ! isolated splenic vein thrombosis

[Term]
id: MONDO:0035009
name: isolated mesenteric vein thrombosis
subset: ordo_disease
is_a: MONDO:0015245 {source="Orphanet:583861"} ! obsolete rare intestinal disease
is_a: MONDO:0000001 {source="Orphanet:583861"} ! disease or disorder
xref: Orphanet:583861 {source="MONDO:equivalentTo"} ! isolated mesenteric vein thrombosis
```

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #2279

### CODE REVIEW

**Comment by matentzn** [comment]
I am sorry I cant review this :/ @cmungall will have to; or I need some guidance what to do!

**Comment by nicolevasilevsky** [comment]
to do: 
- [ ] add definition to primary desmosis coli

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -420307,6 +420309,73 @@ intersection_of: MONDO:0001328 ! thyroid hormone resistance syndrome
...
+subset: ordo_disease
+xref: OMIM:616115 {source="Orphanet:576349"}
+xref: Orphanet:576349 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000001 {source="Orphanet:576349"} ! disease or disorder
+is_a: MONDO:0017370 {source="Orphanet:576349"} ! autoinflammatory syndrome with skin involvement
+is_a: MONDO:0017953 {source="Orphanet:576349"} ! hereditary periodic fever syndrome
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/16412 {source="Orphanet:576349"} ! NLRC4
+
+[Term]
+id: MONDO:0034976
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:420341]: need an iatrogenic DP - can you do @nicolevasilevsky - but axioms can be added later

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -420307,6 +420309,73 @@ intersection_of: MONDO:0001328 ! thyroid hormone resistance syndrome
...
+name: primary desmosis coli
+def: "A rare intestinal disease characterized by congenital partial or complete lack of the collagen mesh network in the intestinal wall, resulting in hypoperistalsis or aperistalsis. The enteric nervous system is normal or near-normal in the affected areas, although hypo- and dysganglionosis may be found in some proximal segments of the colon and/or small bowel. Patients present with chronic intractable slow transit constipation." [Orphanet:565641]
+subset: ordo_disease
+xref: ICD10:K59.9 {source="Orphanet:565641/ntbt"}
+xref: Orphanet:565641 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005020 {source="Orphanet:565641"} ! intestinal disease
+
+[Term]
+id: MONDO:0034849
+name: laminin subunit alpha 2-related limb-girdle muscular dystrophy R23
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:420323]: what is this? is this a standard naming convention? is this just an unusual name for https://www.omim.org/entry/618138

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -420307,6 +420309,73 @@ intersection_of: MONDO:0001328 ! thyroid hormone resistance syndrome
...
+subset: ordo_disease
+xref: Orphanet:565837 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015152 {source="Orphanet:565837"} ! autosomal recessive limb-girdle muscular dystrophy
+is_a: MONDO:0016149 {source="Orphanet:565837"} ! qualitative or quantitative defects of merosin
+
+[Term]
+id: MONDO:0034972
+name: NLRC4-related familial cold autoinflammatory syndrome
+subset: ordo_disease
+xref: OMIM:616115 {source="Orphanet:576349"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:420333]: what is the semantics of the xref? Looks like equiv? https://www.omim.org/entry/616115

in which case merge with MONDO:0014498

**Review by cmungall — APPROVED** [review]
go ahead and merge but make sure comments are taken care of in separate tickets

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:420323]: Oh, I didn't catch that. I'll add this as an xref on the OMIM term- thanks!

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:420333]: will do!

**Pull request merged**